offer-manager-app/src/main/resources/offer-template/offer.html

```diff
@@ -577,6 +632,15 @@ <h2>Quotation Details</h2>
                             -1,690.00
                         </div>
                     </div>
+                    <div class="row total-costs" id="total-before-vat">
+                        <div class="col-6"></div>
+                        <div class="col-4 cost-summary-field">
+                            Total before VAT:
+                        </div>
+                        <div class="col-2 price-value" id="total-before-vat-value">
+                            ?
+                        </div>
```

Comment:
NIT: Since the template numbers are random anyway you can just replace the ? with a number. 
MAJOR NIT: I'd rather have a correct calculation in the template for showcasing purposes, meaning that the overhead cost, discount, vat and total numbers add up as expected but this is not part of this PR.

Reply:
Tbh this is not part of the release branch. This should have been adjusted when the html content has been changed. Imo the release branch should only change stuff related to the release e.g changelog or versioning.

Reply:
You can open an issue to keep track of that :)